app/assets/templates/aspect_create_modal_tpl.jst.hbs

```diff
@@ -42,7 +42,7 @@
         <div class="btn btn-default" data-dismiss="modal" aria-hidden="true">
           {{ t "cancel" }}
         </div>
-        <div class="btn btn-primary">
+        <div class="btn btn-primary" data-dismiss="modal" data-backdrop="false">
```

Comment:
AFAIK `data-backdrop="false"` doesn't do anything here. You can add it to buttons which open the modal to disable the backdrop.